Library/Homebrew/tap.rb

```diff
@@ -472,7 +477,7 @@ def command_dir
   def command_file?(file)
     file = Pathname.new(file) unless file.is_a? Pathname
     file = file.expand_path(path)
-    file.parent == command_dir && file.basename.to_s.match?(/^brew(cask)?-/) &&
+    file.parent == command_dir && file.file? &&
```

Comment:
Might be nice to reuse some logic from https://github.com/Homebrew/brew/blob/master/Library/Homebrew/commands.rb here?

Reply:
With the revamp I figured it was OK to nuke this method entirely.